azurerm/resource_arm_function_app.go

```diff
@@ -199,6 +199,26 @@ func resourceArmFunctionApp() *schema.Resource {
 							Optional: true,
 							Computed: true,
 						},
+						"cors": {
```

Comment:
If this is the same schema used in `app_service` could we maybe create a shares schema function here?